inference_perf/config.py

```diff
@@ -124,7 +124,15 @@ class ReportConfig(BaseModel):
 
 class PrometheusClientConfig(BaseModel):
     scrape_interval: int = 15
-    url: HttpUrl = HttpUrl(url="http://localhost:9090")
+    url: Optional[HttpUrl] = None
+    filters: List[str] = []
```

Comment:
Are we setting the existing model name filter for this config when users don't specify any?